x-pack/plugins/apm/server/lib/metrics/metricsFetcher.ts

```diff
@@ -12,7 +12,8 @@ export async function fetchMetrics<T = void>({
   serviceName,
   setup,
   timeseriesBucketAggregations = {},
-  totalAggregations = {}
+  totalAggregations = {},
+  bool = {}
```

Comment:
FWIW I think this function is becoming a bit unwieldy and might need to be turned "inside-out", to just allow other queries to use some helpers to build out the shared pieces now that so much of the internals here are diverging. But for this bug-fix, I left things as simple as possible.